src/praisonai-mcp/praisonai_mcp/__init__.py

```
@@ -70,4 +76,11 @@ def __getattr__(name: str):
     raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
 
 
-__all__ = ["__version__", "MCPServer", "handle_mcp_command", "serve_agents"]
+__all__ = [
+    "__version__",
+    "MCPServer",
+    "handle_mcp_command",
+    "serve_agents",
+    "SESSION_LIST_TOOL",
+    "STATUS_AWAITING_APPROVAL",
+]
```

Ruff (0.16.0)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)